openedx/features/caliper_tracking/tests/expected/video_show_cc_menu.json

```diff
@@ -0,0 +1,42 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Showed",
+  "actor": {
+    "id": "http://localhost:18000/u/verified",
+    "name": "verified",
+    "type": "Person"
+  },
+  "eventTime": "2018-10-23T09:04:49.446Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+      "course_id": "course-v1:ABC+CS103x+2018_T2",
+      "event_source": "browser",
+      "event_type": "video_show_cc_menu",
+      "host": "8433fad63981",
+      "ip": "172.18.0.1",
+      "name": "edx.video.language_menu.shown",
+      "org_id": "ABC",
+      "page": "http://localhost:18000/courses/course-v1:ABC+CS103x+2018_T2/courseware/6d50d03de20a44e690dde8dbd7467917/ab7879d6d17743dcb4f7fdd96949dfe6/2?activate_block_id=block-v1%3AABC%2BCS103x%2B2018_T2%2Btype%40vertical%2Bblock%400e505063819941d0885bf504b5572ec7",
+      "path": "/event",
+      "session": "d6db2ece54e8cf1943c19b68518896d1",
+      "user_id": 8
+    }
+  },
+  "id": "urn:uuid:849b00d0-ab84-4169-8ae7-2f537eeaf56c",
+  "object": {
+    "duration": "PT2M21S",
+    "extensions": {
+      "code": "3_yD_cEKoCk",
+      "id": "e5bd3f7c2a3b4ea78ce01dd990d1b0f1"
+    },
+    "id": "http://localhost:18000/courses/course-v1:ABC+CS103x+2018_T2/courseware/6d50d03de20a44e690dde8dbd7467917/ab7879d6d17743dcb4f7fdd96949dfe6/2?activate_block_id=block-v1%3AABC%2BCS103x%2B2018_T2%2Btype%40vertical%2Bblock%400e505063819941d0885bf504b5572ec7",
+    "type": "VideoObject"
```

Comment:
Isn't type `Frame` more suitable? Because the user is opening a small menu here, not the whole video.